Fixing #73

atomicpages opened the pull request.

atomicpages committed:
Fixing #73

atomicpages committed:
ci(gulp) upgrading gulp and otehr packages

- Moving to gelp 4.x
- Upgrading stylelint and lots of other packages
- Formatting using stylelint in Gulpfile
- Using dart-sass to compile instead of node-sass

atomicpages committed:
feat(:tada:) maintain fork of pretty-checkbox

- Using prettier to format :art:
- Using github actions for build :construction_worker:
- Releasing under scoped package :package:

atomicpages committed:
build(:construction_worker:) whoops, using npm i

atomicpages committed:
refactor(:recycle:)

- Cleaning up README
- Cleaning up package.json

atomicpages committed:
3.0.4

lokesh-coder commented:
hey @atomicpages wow thats a cool upgrade. I see this code was released in `@djthoms/pretty-checkbox` package. Should we push to this library as well?

atomicpages commented:
@lokesh-coder whoops, sorry for the noise. I was planning on maintaining that as a separate fork. I had some other ideas of adding CSS variable support (which would deprecate IE 9/10 and require a polyfill for IE 11). If you're open to this and are interested in a collab LMK 😎 .

Edit: Also happy to merge the current changes in, too.

atomicpages committed:
feat(:pencil:) new docs phase 1

- Moving to docusaurus 2.x
- Re-writing pretty mucg all of the docs

atomicpages committed:
docs(:pencil:) Docs docs docs

- Updating colors
- Finalizing docs
- Giving some love

atomicpages committed:
docs(:pencil:) Docs docs docs

atomicpages committed:
build(:construction_worker:)

- Splitting PR and push builds
- Adding docs check from netlify
- Prepping for netlify deployment

atomicpages committed:
bugfix(:bug) bad yaml syntax

atomicpages committed:
ci(:rocket:) netlify

atomicpages committed:
docs(:pencil:) Updating README

atomicpages committed:
refactor(:zap:)

- Reducing boilerplate in the Gulpfile
- Removing browser sync
- Pushing some changes to the package.json instead
- Removing seemingly unused test.css file

atomicpages committed:
test(:rotating_light:) test scaffolding

atomicpages committed:
Fixing clerical error

atomicpages committed:
feat(:sparkles:) rtl support

Adding new rtl.css file (or rtl.scss) file that can be imported to enable rtl support

atomicpages committed:
3.1.0

atomicpages committed:
Merge branch 'master' of https://github.com/atomicpages/pretty-checkbox into master

atomicpages committed:
docs(:pencil:) adding RTL docs

- Upgrading docusaurus
- Adding rtl docs
- IA reorganization

atomicpages committed:
docs(:pencil:) IA

atomicpages committed:
ci(:green_heart:)

atomicpages committed:
:memo: Adding docsearch to site

atomicpages committed:
:memo:
